A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -88,11 +92,12 @@ var A320_Neo_LowerECAM_Engine;
             switch (_state) {
                 case A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A:
                     {
-                        return "A";
+                        return "A ";
                     }
                 case A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.B:
                     {
-                        return "B";
+                        return " B";
+
```

Comment:
Remove space

Reply:
this space is needed cause it's either AB, A or B, and in the later case is shown wher the B in AB is. Could replace by &nbsp;

Reply:
Means the empty line 100, not the space inside the string literal on line 99